src/keywords.rs

```diff
@@ -491,6 +491,7 @@ define_keywords!(
     INTERSECTION,
     INTERVAL,
     INTO,
+    INVENTORY,
```

Comment:
Can we add test cases to demonstrate the added functionality?

Reply:
Certainly @iffyio. I had been looking through the code for existing tests, but to no avail. So unsure if I should add some cases to the tokenizer tests or in the parser module?

Or is there a more obvious place I am missing?

Reply:
We can probably extend[ this test ](https://github.com/apache/datafusion-sqlparser-rs/blob/main/tests/sqlparser_redshift.rs#L412)to include new scenarios?

Reply:
@iffyio - your hint got me to dive a bit deeper into the codebase. Unfortunately the Redhsift and Databricks VACUUM commands are quite different.

That said, would you accept a PR where I extend this a bit, add a custom statement parser option the databricks dialect and a `VacuumDatabricks` Statement variant?

Reply:
Ah yeah it should possible to support databricks' variant of the vacuum command. But we tend to not have dialect specific statements so that it'll likely need to be part of the existing `Statement::Vacuum` variant in this case, being extended to support the new options